services/apm/apm.service.js

```diff
@@ -41,11 +41,13 @@ class BaseAPMService extends BaseJsonService {
 }
 
 class APMDownloads extends BaseAPMService {
+  static render(downloads) {
```

Comment:
Could we pass `{ downloads }` here instead? I prefer that pattern as it reads and grows well.

Reply:
Shall I switch all the `fetch()` and `render()` methods to use the object destructuring syntax, or just the `render()`s ? I don't have particularly strong opinions on this, I'm just slightly unsure what the tradeoff is in this situation..

Reply:
I think named parameters are great. They make the calling code self-documenting, and allow functions to grow organically with minimal impact at the callsite. As the number of parameters grows past one, naming them reduces the risk of passing the wrong value to a function. Of course JS doesn't have named parameters, but destructuring provides basically the same thing.

So, I'd suggest doing both!